src/auto/device.rs

```diff
@@ -206,9 +207,11 @@ impl Device {
         }
     }
 
-    //pub fn list_axes(&self) -> /*Ignored*/Vec<Atom> {
-    //    unsafe { TODO: call ffi::gdk_device_list_axes() }
-    //}
+    pub fn list_axes(&self) -> Vec<Atom> {
+        unsafe {
+            FromGlibPtrContainer::from_glib_container(ffi::gdk_device_list_axes(self.to_glib_none().0))
```

Comment:
This function need be tested too.
For my default display it returns 2 devices and this function return empty vector, so I can't test it
```rust
let display = gdk::Display::get_default().unwrap();
let devices = display.list_devices();
println!("{}", devices.len());

for device in devices {
    println!("name: {:?}", device.get_name());
    let axes = device.list_axes();
    println!("{}", axes.len());
}
```